Gemfile.lock

```diff
@@ -267,7 +266,7 @@ DEPENDENCIES
   pry-byebug
   pry-rescue
   rails (~> 5.0.0)
```

Comment:
Was this file regenerated properly? This shouldn't be listed as a top level dependency if it's not in the `gemspec` anymore.

Reply:
Yes, the Gemfile specifies a Rails version that we use for local development, it's not part of the gemspec

Reply:
actually, I have updated the appraisals to use railties instead of the full rails, so this should go away now.